python/graphstorm/model/edge_gnn.py

```diff
@@ -180,13 +180,32 @@ def edge_mini_batch_gnn_predict(model, loader, return_proba=True, return_label=F
     labels = {}
     model.eval()
 
+    len_loader = max_num_batch = len(list(loader))
+    barrier()
```

Comment:
We don't need barrier() here. all_reduce by default is synced.

Reply:
I was wondering the same but could not find the exact documentation. Thanks for clarifying.